<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor</class>
 <widget class="QMainWindow" name="editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>editor</string>
  </property>
  <property name="windowIcon">
   <iconset theme="postersafari">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QGroupBox" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Plakat bearbeiten</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="userDataBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Benutzerdefinierte Einträge</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="userDataEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="postrDataBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Assoziiertes Dokument</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTextEdit" name="postrDataEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="buttonReload">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Neu laden</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonFollowStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonProcess">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Verarbeiten</string>
         </property>
         <property name="icon">
          <iconset theme="postersafari">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonFollowStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSaveDB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Speichern</string>
         </property>
         <property name="icon">
          <iconset theme="document-save">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonFollowStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDokument">
    <property name="title">
     <string>Doku&amp;ment</string>
    </property>
    <addaction name="actionProcess"/>
    <addaction name="separator"/>
    <addaction name="actionSaveJson"/>
    <addaction name="actionSaveDB"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuBild">
    <property name="title">
     <string>Bi&amp;ld</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRotate90"/>
    <addaction name="actionRotate270"/>
    <addaction name="actionRotate180"/>
   </widget>
   <addaction name="menuDokument"/>
   <addaction name="menuBild"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="documentListDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="documentListDockContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="documentList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="documentImageDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="documentImageDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="documentImageView"/>
     </item>
     <item>
      <widget class="QSlider" name="documentImageViewZoom">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="documentImageView2"/>
     </item>
     <item>
      <widget class="QSlider" name="documentImageView2Zoom">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveJson">
   <property name="text">
    <string>Als json &amp;speichern...</string>
   </property>
  </action>
  <action name="actionSaveDB">
   <property name="text">
    <string>&amp;In Datenbank speichern</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Neu laden</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Aus Datenbank löschen</string>
   </property>
  </action>
  <action name="actionRotate90">
   <property name="text">
    <string>&amp;Um 90° nach rechts drehen</string>
   </property>
  </action>
  <action name="actionRotate270">
   <property name="text">
    <string>Um &amp;90° nach links drehen</string>
   </property>
  </action>
  <action name="actionRotate180">
   <property name="text">
    <string>Um &amp;180° drehen</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="text">
    <string>&amp;Verarbeiten</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSaveDB</sender>
   <signal>clicked()</signal>
   <receiver>actionSaveDB</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonProcess</sender>
   <signal>clicked()</signal>
   <receiver>actionProcess</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
